internal_ws/yktrace/src/lib.rs

```diff
@@ -23,6 +25,7 @@ use sir::SirTrace;
 use ykpack::Local;
 
 // In TIR traces, the argument to the interp_step is always local #1.
+// XXX this constant could be used in quite a few places throughout the repo!
```

Comment:
Fixed in 01a96d0f1a803e5ca4d78eb5ea31bec05847e753.

Reply:
Has this been merged already?